source/NPC.cpp

```diff
@@ -811,7 +811,7 @@ void NPC::DoActions(const ShipEvent &event, bool newEvent, PlayerInfo &player, U
 					return it != shipEvents.end() && (it->second & requiredEvents) && !(it->second & excludedEvents);
 				}))
 		{
-			it->second.Do(player, ui, caller);
+			it->second.Do(player, ui, caller, trigger == Trigger::BOARD ? event.Target() : nullptr);
```

Comment:
Thoughts on just always passing the event target in? I don't see any particular reason to mechanically limit this behavior to just boarding NPC actions.

Reply:
I decided not to do that but I don't have any particularly strong reasoning for it.